regression/contracts/variant_multidimensional_not_decreasing_fail/test.desc

```diff
@@ -0,0 +1,16 @@
+CORE
+main.c
+--apply-loop-contracts
+activate-multi-line-match
+^\[main.1\] .* Check loop invariant before entry: SUCCESS$
+^\[main.2\] .* Check that loop invariant is preserved: SUCCESS$
+^\[main.3\] .* Check decreases clause on loop iteration: FAILURE$
+^.*1 of 3 failed \(2 iterations\)$
+^VERIFICATION FAILED$
+^EXIT=10$
+^SIGNAL=0$
+--
+--
+This test fails because the given multidimensional decreases clause does not 
+monotonically decrease. A mistake is in the second component of the
+decreases clause: j. It should instead be N - j. 
```

Comment:
Can we also add another loop in the function where the first component is not strictly monotonically decreasing? Just to be sure that we are indeed checking all components of tuples (well at least of size 2, in this case).

Reply:
Got it.